account-kit/smart-contracts/src/ma-v2/account/modularAccountV2.ts

```diff
@@ -110,12 +143,64 @@ export async function createModularAccountV2(
     deferredAction,
   } = config;
 
+  const signer = "signer" in config ? config.signer : undefined;
+
+  const credential = "credential" in config ? config.credential : undefined;
+
+  const getFn = "getFn" in config ? config.getFn : undefined;
+
+  const rpId = "rpId" in config ? config.rpId : undefined;
+
+  const client = createBundlerClient({
+    transport,
+    chain,
+  });
+
   const accountFunctions = await (async () => {
     switch (config.mode) {
+      case "webauthn": {
+        if (!credential)
+          throw new Error("Missing params for MAV2 webauthn mode");
```

Comment:
nit: let's make this a strongly typed error since that will allow us to also link to docs in the future